README.md

```diff
@@ -106,52 +125,31 @@ status changed: offline
 */
 ```
 
+#### [Run example](https://runkit.com/zerobias/effector-storages-and-events)
+
+<hr />
+
 ## Demo
 
 > [![Edit Effector-react example](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/vmx6wxww43) Basic example
 
 > [![Edit Effector-react example](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/1y6n4r6o57) SSR example
 
-## Documentation
-
-- [Glossary](https://github.com/zerobias/effector/wiki/Glossary)
-- [API Reference](https://github.com/zerobias/effector/wiki/API-Reference)
-
-## Usage
-
-```js
-import {createStore, createEvent} from 'effector'
-import {createComponent} from 'effector-react'
+### More examples/demo you can check [here]()
 
-const increment = createEvent('increment')
-const decrement = createEvent('decrement')
-const resetCounter = createEvent('reset counter')
+## Core concept
 
-const counter = createStore(0)
-  .on(increment, state => state + 1)
-  .on(decrement, state => state - 1)
-  .reset(resetCounter)
+![Core](https://github.com/zerobias/effector/blob/master/modules.png)
```

Comment:
I don't think that this should be shown on the main page; I suggest adding this to the readme in src, for those interested in the source code